cognite/client/_api/templates.py

```diff
@@ -47,7 +47,7 @@ def graphql_query(self, external_id: str, version: int, query: str) -> GraphQlRe
                 >>>        }
                 >>>    }
                 >>>    '''
```

Comment:
User also mentioned missing some []’ for datapoints, cannot find your fix for the mentioned typo here